app/views/lks/form.html

```diff
@@ -0,0 +1,28 @@
+{% extends "lks-layout.html" %}
+{% block pageTitle %}
+Home - HEE
+{% endblock %}
+
+{% block content %}
+
+<form class="form" method="post" name="newsletter">
+    <h2>Subscribe to our newsletter</h2>
+
+    <div id="page0">
+        <div class="form-field form-field-error">
```

Comment:
Re-use NHSUK design system components:
https://service-manual.nhs.uk/design-system/components/text-input

And also look at wrapping these sections with fieldsets:
https://service-manual.nhs.uk/design-system/components/fieldset